### DISCUSSION TITLE
Freelance Tips

### DISCUSSION THREAD

**Discussion by AllanRegush** [EddieHubCommunity/support#261] [Q&A]
Does anyone have tips for freelancing?

**Comment by eddiejaoude** [comment]
First rule of freelancing, start 🤓 (I always wanted to say that 😂 ). Ok, that is easier said than done I know.

Having been a freelancer for over 10 years, I can not recommend it enough. However, at the beginning it is definitely more difficult.

## General 
- It is a marathon not a sprint (so cheesy Eddie 🤦‍♂️ ), but what I mean is, you dont want to give up before things start picking up, so I recommend trying to do it on the side with a current job or have some savings on the side

## Accounts
- Each country is different regarding accounts, but I highly recommend getting an accountant - will cost you money but will save time and money in the long run
- Dont spend money that will be due for tax, I see people get themselves in a right mess - keep it to one side

## Clients
- Under promise and over delivery, that way people will sing your praises to their friends and colleagues
- Contact and connect with agencies who have a lot of work on, they can share their work load - you will make less money with this, but helps to grow your network, confidence and some savings
- There are platforms for freelancers, but I hear they are a race to the bottom on pricing - might be a place to start though and can break away later

## Expenses
- Taxes mentioned above
- Advertising - you might want to put a local ad somewhere 
- Insurance - probably never needed but better to have

## Time
- Log time spend on admin, accountants, with each client on calls etc
- Allow time for admin, accounts, finding work, giving quotes etc

I am sure I have missed many things, any questions let me know, I will think more tomorrow and update this 🤓

**Reply by stemount** [reply]
> There are platforms for freelancers, but I hear they are a race to the bottom on pricing - might be a place to start though and can break away later.

Huge +1 on this, however it doesn't hurt to bid for jobs on these or even build a profile.

There are (in many countries) startups doing Freelancer.com (as jobs, not fixed projects) for {insert country here} which actvually will generally yield better work than one-size-fits-all-somebody-will-do-it-for-20-usd-for-their-portfolio platforms.

Contracting vs Freelancing is a whole different ballgame, the former suggests you are in some capacity contracted for a duration of time rather than the latter which sometimes (not always) suggests quoting a fixed price for services. It's sometimes a higher pay-off doing the former if you love juggling work....

**Reply by eddiejaoude** [reply]
true! there is always confusing between contract and freelance, this explains it well 👍

**Comment by Aureaclavel** [comment]
Hi, I am a newbie and eager to have my first client. I hope with this week now. I already made my website but I still have to do more on it. Thank you for the tips.

https://professionalgroupdotorg.wordpress.com/

**Reply by eddiejaoude** [reply]
Looks good 👍 

Some suggestions:

- Add your GitHub link to your website
- GitHub profile to have a profile picture, description, website link etc
- Social links on website do not go to your account

I hope that helps 🤓

**Comment by emmysteven** [comment]
Don't start any work without a contract being signed by you and your client
I learned the hard way

**Reply by eddiejaoude** [reply]
Yeah good point 👍 and try to get some money upfront 25-50%, I find people are more committed to complete the project by providing information / feedback etc
